main.js

```diff
@@ -0,0 +1,23 @@
+const hamburger = document.querySelector('.menu-button');
+const menuClose = document.querySelector('.menu-close');
+const menuOverlay = document.getElementById('menu-overlay');
+
+const displayMenuList = () => {
+  hamburger.style.display = 'none';
+  menuOverlay.style.display = 'block';
+};
+
+const closeMenuList = () => {
+  menuOverlay.style.display = 'none';
+  hamburger.style.display = 'block';
+};
+
+hamburger.addEventListener('click', displayMenuList);
+menuClose.addEventListener('click', closeMenuList);
+
+document.querySelectorAll('.menu-item-list').forEach((n) => {
+  n.addEventListener('click', () => {
+    menuOverlay.style.display = 'none';
+    hamburger.style.display = 'block';
+  });
+});
```

Comment:
- [ ] Kindly display the hamburger icon only for the mobile menu. Remove the hamburger icon from the desktop version as stated in the attached loom video.
- [ ] Kindly follow the Figma design for the mobile popup menu. Use the backdrop with the background colour: `background: #6070FF;` to make it as close to the design as possible as shown in the attached loom video.

Reply:
Thanks @sja-thedude for the review and for the specific directions 👏